docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md

```diff
@@ -95,6 +96,37 @@ To create an export configuration, do the following:
 1. Enter your installation token, access ID, and access key.
 1. Optionally, click **Download** to download the Sumo Logic dashboard template. After you install the [YugabyteDB app](https://help.sumologic.com/docs/get-started/apps-integrations/) (coming soon) in your Sumo Logic account, you can import this JSON format template and use it as a starting point for visualizing your cluster data.
 1. Click **Test Configuration** to make sure your connection is working.
+1. Click **Create Configuration**.
+
+  {{% /tab %}}
+
+  {{% tab header="Prometheus" lang="prometheus" %}}
+
+Prometheus integration is {{<badge/tp>}} and only available for clusters deployed on AWS.
+
+The Prometheus integration requires the following:
+
+- Prometheus instance
+  - deployed in a VPC on AWS
+  - [OLTP Receiver](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver) feature flag enabled
+  - publically-accessible endpoint URL; the DNS for the endpoint must be in a public hosted zone in AWS.
+- YugabyteDB cluster from which you want to export metrics
+  - the cluster is [deployed in VPCs](../../cloud-basics/cloud-vpcs/cloud-add-vpc/) on AWS
+  - each region VPC is peered with the VPC hosting Prometheus. See [Peer VPCs](../../cloud-basics/cloud-vpcs/cloud-add-vpc-aws/).
+  - VPCs have the following Inbound Security Group rules:
```

Comment:
These inbound security group rules apply to the VPC in which Prometheus is hosted, not the yugabyte cluster vpc. We are exporting metrics to the customer VPC, hence customer VPC should have inbound rules defined